sprint_tasks/MSI_2/MSI_2_send_message_ms_teams_dag.py

```diff
@@ -0,0 +1,163 @@
+import datetime
+import logging
+import random
+from abc import ABC, abstractmethod
+
+import pendulum
+import pymsteams
+import requests
+from airflow import DAG
+from airflow.operators.empty import EmptyOperator
+from airflow.operators.python import PythonOperator
+from requests.exceptions import ConnectionError, HTTPError
+
+logging.basicConfig(
```

Comment:
Airflow has a different logging format setup. Please view https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/logging-monitoring/advanced-logging-configuration.html